Libplanet/Net/Swarm.BlockSync.cs

```diff
@@ -155,11 +155,11 @@ CancellationToken cancellationToken
                     lastUpdated = DateTimeOffset.UtcNow;
                     lastTip = BlockChain.Tip;
                 }
-                else if (lastUpdated + tipLifespan > DateTimeOffset.UtcNow)
+                else if (lastUpdated + tipLifespan < DateTimeOffset.UtcNow)
```

Comment:
Would it be difficult to add a regression test for this bug?

Reply:
Well, it's probably possible to do a unit test by elevating the access level to `internal` and directly calling the method, but I'm not sure about a regression test. 😗